packages/cardano-services/src/Http/HttpServer.ts

```diff
@@ -30,6 +30,29 @@ export class HttpServer extends RunnableModule {
     this.#config = config;
     this.#dependencies = { logger, ...rest };
   }
+
+  async #getServicesHealth() {
+    const servicesHealth: ServiceHealth[] = await Promise.all(
+      this.#dependencies.services.map((service) =>
```

Comment:
Maybe it would be nice to expose each service separately?

Reply:
We decided to go with a single reduced server health metric as the goal here is to just signal when the `http-server` should be restarted. There's a more appropriate HTTP endpoint `/health` that returns a structured JSON response with granular information.